qt/applications/workbench/CMakeLists.txt

```diff
@@ -49,6 +56,11 @@ if(WIN32)
     # Write qt.conf.BUILD_TYPE file and get resources file name
     if(${BUILD_TYPE} STREQUAL "build")
       file(WRITE ${_qt_conf_file} "[Paths]\nPrefix = ${_qt5_dir}\n\n[Platforms]\nWindowsArguments = dpiawareness=1\n")
+      file(
```

Comment:
Could I ask why it was necessary to add this?
My understanding of the code was that the file was written to disk and compiled inside the `resources.py` using the `add_custom_command` below?

Reply:
If I remember correctly, this was added so that the `MantidHelpWindowTest.h` could start a QApplication. It was looking in the bin folder for this `qt.conf` file but couldn't find it. Maybe it is looking in the wrong location for some reason?

Reply:
Yes I see now. The unit test executable now ends up in the bin directory and it looks there. Thanks for the explanation.